src/main/analytics/webdav-handlers.ts

```diff
@@ -10,27 +10,52 @@ function subscribeToFlowEvents(ipc: IpcWebdavFlow) {
   ipc.on('WEBDAV_FILE_DELETED', (_, payload) => {
     const { name, type, size } = payload;
 
+    broadcastToWindows('sync-info-update', {
+      action: 'DELETED',
+      name: name + '.' + type,
+    });
+
     trackWebdavEvent('Delete', {
       name,
       type,
       size,
     });
   });
 
+  ipc.on('WEBDAV_FILE_DOWNLOADING', (_, payload) => {
+    const { name, type, progress } = payload;
+
+    broadcastToWindows('sync-info-update', {
+      action: 'DOWNLOADING',
+      name: name + '.' + type,
+      progress,
+    });
+  });
+
   ipc.on('WEBDAV_FILE_DOWNLOADED', (_, payload) => {
-    const { name, type, size, uploadInfo } = payload;
+    const { name, type, size, processInfo } = payload;
+
+    broadcastToWindows('sync-info-update', {
+      action: 'DOWNLOADED',
+      name: name + '.' + type,
```

Comment:
what if file doesn't has type? we display 'filename.' ?

Reply:
nice catch, I've added a new property nameWithExtension for a better management